pkg/install/install.go

```diff
@@ -253,7 +253,10 @@ func (i *Installer) runSteps(ctx context.Context, steps []interface{}) error {
 				if _, ok := err.(*api.CloudError); !ok {
 					err = fmt.Errorf("%s: %s", runtime.FuncForPC(reflect.ValueOf(step).Pointer()).Name(), err)
 				}
+				i.log.Warnf("ran step but got an error (%s)", err)
```

Comment:
isn't this error printed down the track? I think this will make the same error to get printed twice

Reply:
agreed

Reply:
where do you see this? This one is for steps, the one further down below is for conditions